Resources/views/CRUD/display_boolean.html.twig

```diff
@@ -0,0 +1,32 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{%- spaceless %}
+    {% if value %}
+        {% set text = 'label_type_yes'|trans({}, 'SonataAdminBundle') %}
+    {% else %}
+        {% set text = 'label_type_no'|trans({}, 'SonataAdminBundle') %}
+    {% endif %}
+
+    {% if field_description.options.inverse is defined and field_description.options.inverse %}
+        {% set isSuccess = not value %}
+    {% else %}
+        {% set isSuccess = value %}
+    {% endif %}
+
+    {% if isSuccess %}
+        {% set class = 'label-success' %}
+    {% else %}
+        {% set class = 'label-danger' %}
+    {% endif %}
+
+    <span class="label {{ class }}">{{ text }}</span>
+{% endspaceless -%}
```

Comment:
Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.